lib/teleterm/clusters/cluster_access_requests.go

```diff
@@ -174,11 +145,8 @@ func (c *Cluster) ReviewAccessRequest(ctx context.Context, req *api.ReviewAccess
 		}
 
 		updatedRequest, err = authClient.SubmitAccessReview(ctx, reviewSubmission)
-		if err != nil {
-			return trace.Wrap(err)
-		}
 
-		return nil
+		return err
```

Comment:
In this case, there are multiple calls that can return `err` within a single function passed to `addMetadataToRetryableError`. It'd be best to wrap each of them so that we know exactly which line returned the error.

```suggestion
		return trace.Wrap(err)
```